api/src/test/java/edu/cornell/mannlib/vitro/webapp/controller/accounts/UserAccountsSelectorTest.java

```diff
@@ -33,6 +32,9 @@ public class UserAccountsSelectorTest extends AbstractTestClass {
 
 	private static final String NS_MINE = "http://vivo.mydomain.edu/individual/";
 
+    private static final UserAccountsOrdering DEFAULT_ORDERING = new UserAccountsOrdering(
```

Comment:
I suggest to change name of this static attribute to ACCOUNTS_ORDERING. It is not default ordering anymore (it is now last login, descending).

Reply:
Done